node/internal.ts

```diff
@@ -426,7 +427,7 @@ export function _which(tool: string, check?: boolean): string {
 function _tryGetExecutablePath(filePath: string, extensions: string[]): string {
     try {
         // test file exists
-        let stats: fs.Stats = fs.statSync(filePath);
+        let stats: fs.Stats = fs.statSync(_unTildify(filePath));
```

Comment:
let me add some tests that deal with these conditions and take it from there. The regex was taken directly from that library referenced by many in the Node issue.

Please stand by....